verticapy/tests/vModel/test_tools.py

```diff
@@ -36,8 +36,9 @@
 from verticapy.learn.decomposition import *
 from verticapy.learn.preprocessing import *
 from verticapy.learn.tsa import *
-from verticapy.learn.tools import does_model_exist, load_model, matrix_rotation
+from verticapy.learn.tools import does_model_exist, load_model
 from verticapy.sql.create import create_verticapy_schema
+import verticapy.machine_learning.memmodel.decomposition as mm
```

Comment:
previously you imported memmodel as "mm". But here you are importing decomposition as "mm".
please check. For consistency, maybe change name to "mm_decomp" or only "decomp"?